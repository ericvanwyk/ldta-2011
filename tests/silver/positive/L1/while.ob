MODULE While;

VAR i,x,y: INTEGER;

BEGIN
  x := 2;
  y := 3;
  i := 0;
  WHILE i < 3
  DO
    x := x * y;
    y := y * x;
    i := i + 1
  END;
  
  Write(x); WriteLn;
  Write(y); WriteLn;
  Write(i); WriteLn;
END While.
