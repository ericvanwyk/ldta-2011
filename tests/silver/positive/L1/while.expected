209952
408146688
3
